Modules/mathmodule.c

```diff
@@ -3234,16 +3328,24 @@ math_prod_impl(PyObject *module, PyObject *iterable, PyObject *start)
 static PyObject *
 perm_comb_small(unsigned long long n, unsigned long long k, int iscomb)
 {
+    static const unsigned char fast_comb_limits1[] = {
+        127, 127, 127, 127, 127, 127, 127, 127,  // 0-7
```

Comment:
Maybe set first two or three items to 0? `k` always `> 0`, and for `k = 1` the second fast path may be faster.